### PR TITLE
[Mellanox] Add system health configuration file for Mellanox platforms 

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
**- Why I did it**

The new feature system health support a platform based configuration file. Add configuration files for all Mellanox platform.

**- How I did it**

Add a configuration file for SN2700, other platform will use a soft link to it.

**- How to verify it**

Manual test on SN2700, SN2410, SN4600C

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: device/mellanox/x86_64-mlnx_msn2700-r0/system_health_monitoring_config.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "services_to_ignore": [],
+    "devices_to_ignore": ["psu.voltage"],
+    "external_checkers": [],
+    "interval": 60,
```
Comment (keboliu) [inline device/mellanox/x86_64-mlnx_msn2700-r0/system_health_monitoring_config.json:5]: polling_interval?

File: device/mellanox/x86_64-mlnx_msn2700_simx-r0/system_health_monitoring_config.json [hunk 2]
```diff
@@ -0,0 +1 @@
+../x86_64-mlnx_msn2700-r0/system_health_monitoring_config.json
```
Comment (keboliu) [inline device/mellanox/x86_64-mlnx_msn2700_simx-r0/system_health_monitoring_config.json:1]: for simx platform, we need a different configuration file, all devices shall be ignored.

Comment (Junchao-Mellanox) [reply device/mellanox/x86_64-mlnx_msn2700_simx-r0/system_health_monitoring_config.json:1]: sure

File: device/mellanox/x86_64-mlnx_msn2700-r0/system_health_monitoring_config.json (hunk 1 above)
Comment (Junchao-Mellanox) [reply device/mellanox/x86_64-mlnx_msn2700-r0/system_health_monitoring_config.json:5]: sure

File: device/mellanox/x86_64-mlnx_msn2700_simx-r0/system_health_monitoring_config.json (hunk 2 above)
Comment (keboliu) [reply device/mellanox/x86_64-mlnx_msn2700_simx-r0/system_health_monitoring_config.json]: one more thing, simx platform doesn't have LED, will it cause error ?

Comment (Junchao-Mellanox) [reply device/mellanox/x86_64-mlnx_msn2700_simx-r0/system_health_monitoring_config.json]: No, it won't

**Comment by Junchao-Mellanox** [comment]
Community PR: https://github.com/Azure/sonic-buildimage/pull/4834

**Pull request closed without merging**